spec/connect/errors_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'ostruct'
+
+describe SUSE::Connect::ApiError do
+  let(:with_http_message) { OpenStruct.new(http_message: 'foo', code: 1337) }
+
+  let(:with_error) do
+    OpenStruct.new(
+      http_message: 'foo',
+      body: { 'error' => 'bar' }
+    )
+  end
+
+  let(:with_localized_error) do
+    OpenStruct.new(
+      http_message: 'foo',
+      body: { 'error' => 'bar',
+              'localized_error' => 'baz' }
+    )
+  end
+
+  describe '.code' do
+    it 'returns the http status code' do
+      expect(described_class.new(with_http_message).code).to eq(1337)
+    end
+  end
+
+  describe '.message' do
+    it 'returns the http message' do
+      expect(described_class.new(with_http_message).message).to eq 'foo'
```

Comment:
I would use parentheses here

Reply:
TBH we have code that calls `eq` with and without parenthesis all over the code. Not sure if it's worth changing it.
If you'd like to make it a "standard" then it would be a good topic to bring up in the team meeting.
But without a Rubocop rule that checks this, it's really annoying as "a hooman" to care for style problems =/